views/homework/components/overview_card_template.hbs

```diff
@@ -12,25 +12,39 @@
             {{$t "homework.text.noSubmissionDateSet"}}
             {{/if}}
         </div>
+        {{#if this.stats}}
+            <a class="content" href="{{this.url}}#activetabid=submissions"
+                aria-label="Abgaben der Aufgabe: '{{{escapeHtml this.name}}}'">
```

Comment:
aria-label text should come from de/en.json

Reply:
wasnot ther ebrefore the fix, should be a follow up.